### DISCUSSION TITLE
Option to automatically start the new round, and the option to show the last round's stats while doing the new round

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#986] [Ideas] [answered]
This is one of things that [Keybr](https://www.keybr.com) does that feels better. They show your last round's main statistics like wpm and errors while you play the next round. Or maybe theirs feel like there is no round at all, just consistently typing. This makes you have a consistent and uninterrupted keyboard typing experience. 

But it wouldn't make a lot of sense to do a round without being able to see your stats, so I suggest that the stats be shown while doing the next round.

**Comment by Miodec** [comment, marked as answer]
Would require a significant UI rewrite, probably not going to happen sorry. You could try using the infinite tests or zen if you want to type uninterrupted.

**Reply by ghost** [reply]
You already have an option to show live wpm and accuracy during the round, so why can't new elements be added to also show last round's wpm and accuracy, wouldn't that just reuse the existing UI? If that is too bloated, then they could be made under "hide elements" in settings. I also get that last round wpm and number of errors might not be very useful to add to hide elements, but there is a timer for for timed tests, and that also is the number of words in progress tests. You know, the more elements the merrier.

Even without that, skipping the end of round stats could be useful under behavior in the settings. For example if you want to write quotes and race against a custom caret moving at a specific speed, and don't care that much about your stats.

**Reply by ghost** [reply]
In any case infinite tests is pretty close to what I was looking for. Thanks for answering!
